filesystems/s3/src/main/scala/cromwell/filesystems/s3/S3PathBuilder.scala

```diff
@@ -130,12 +130,13 @@ object S3PathBuilder {
                       configuration: S3Configuration,
                       options: WorkflowOptions,
                       storageRegion: Option[Region]): S3PathBuilder = {
+    AmazonS3ClientProvider.init(credentials, storageRegion.getOrElse(Region.US_EAST_2))
```

Comment:
If I understand everything correctly, creation of the `S3Client` is happens before these steps.
Therefore, I think, it does not depends on authorization type. But to be completely sure, I will test it with `default` auth method tomorrow, because before I tested this only with `custom_keys` auth type.

Reply:
As I see, everything is OK with both `default` and `custom_keys` auth types, but I would appreciate more testing from your side :smiley: